### ISSUE TITLE
EV charger values missing

### ISSUE THREAD

**Issue by MonsterEnergy-wtf** [snitzelweck92/homeassistant_alphaess_modbus_tcp#17]
Hi devs,

first of all thanks for this implementation. The cloud API from AlphaESS really sucks and I'm really happy to see the data from solar and grid consumption in HA.
I just have one little request: is it possible to see also the consumption by the EV charger?
I have one from AlphaESS and at least the app from AlphaESS shows me the consumption. 
Is it possible to get values related to this from the inverter?
Currently I'm using a SMILE-T10-HV with a SMILE-G3-EVCT11

I also tried to use EVCC as a intermediate software and got values from it, but the received values are messed up somehow and also the charger seems not to be able to work with EVCC via ocpp. 

So long story short: I would be happy to see only the consumption of the charger somewhere. No matter if this is nice or not. 

If this is not possible, just close this ticket. No worries.
